tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -282,3 +287,18 @@ def run_fio_on_pvcs(self, pvc_size):
             performance_lib.write_fio_on_pod(objs, file_size_mb_str)
 
         return total_files_size
+
+    def test_bulk_clone_performance_results(self):
+        """
+        This is not a test - it is only check that previous test ran and finish as expected
+        and reporting the full results (links in the ES) of previous tests (3)
```

Comment:
@ramkiperiy I believe here also it has to be changed into 2 tests. But this could be changed in future PRs since it is docstring.

Reply:
updated the comments, resolving the comment :)